data/transition-sites/gds_alphagov_alert-preview.yml

```diff
@@ -0,0 +1,11 @@
+---
+site: gds_alphagov_alert-preview
+whitehall_slug: government-digital-service
+homepage: https://alert.preview.publishing.service.gov.uk/
```

Comment:
Could you drop the trailing slashes on all the `homepage` and `global` URLs? The request path will be appended to the `global` URL so we'll end up with double slashes otherwise.